<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CopyFinder</class>
 <widget class="QMainWindow" name="CopyFinder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CopyFinder</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #1E656D;
color: #F1F3C3;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">font: 15pt &quot;Trebuchet MS&quot;;</string>
          </property>
          <property name="text">
           <string>Choose Directory</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="chooseDirectoryButton">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Trebuchet MS&quot;;
background-color: #00293C;
border-radius: 3px;
height: 21px;</string>
          </property>
          <property name="text">
           <string>Choose</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="styleSheet">
         <string notr="true">QTreeView {
	font: 12pt &quot;Trebuchet MS&quot;;
	color: #00293C;
	background: #F1F1F2;
}

QTreeView::item:disabled {
	color: #92AAC7;
}</string>
        </property>
        <attribute name="headerMinimumSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="headerStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Filename</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Size</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>25</number>
        </property>
        <item>
         <widget class="QPushButton" name="findCopiesButton">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Trebuchet MS&quot;;
background-color: #00293C;
border-radius: 3px;
height: 21px;</string>
          </property>
          <property name="text">
           <string>Find Copies</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelButton">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Trebuchet MS&quot;;
background-color: #00293C;
border-radius: 3px;
height: 21px;</string>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar {
border: 1px solid black;
color: #1E1F26;
text-align: center;
padding: 1px;
border-top-left-radius: 7px;
border-bottom-left-radius: 7px;
background: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #fff,
stop: 0.4999 #eee,
stop: 0.5 #ddd,
stop: 1 #eee );
width: 15px;
}
QProgressBar::chunk {
background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0,
stop: 0 #0000ff,
stop: 1 #ff0000 );
border-top-left-radius: 7px;
border-bottom-left-radius: 7px;
border: 1px solid black;
}</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deleteCopiesButton">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Trebuchet MS&quot;;
background-color: #00293C;
height: 21px;
border-radius: 3px;
margin-top: 5px;
</string>
      </property>
      <property name="text">
       <string>Delete Copies</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionScan_Directory"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_us"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionScan_Directory">
   <property name="text">
    <string>Scan Directory</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout_us">
   <property name="text">
    <string>About us</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
